multi-arch-builders/builder-common.bu

```diff
@@ -27,7 +27,8 @@ passwd:
         - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQC7znT80FNnSXdVbfp6oOgBQBe2Df/RFzqSTb4FOM9HNo09nfB0L4JHaFpQsg5UO6TX95+Z069XUc8Q0xNhdzURZWDJQY/FrbV+2EtqfDQNnLZybFTXtvOBVbKveMwBtYBEiXxfJiWUv3RCQ/oEpDYGSQ5ZGoqjOVHoBf5FXimp8GiipT+WjZbTjq24tM1ZSRxaSZ94C+MgJ7cbCyJ6JTlY01goX9tyeR/C0l7A3RX1a4v0kt/r4Q8jT5YgUzqm+gz9CKCEukSvIUVBqrhPWvjIdPLvq9lyxa4PNrqkkQvjuJgiPGuC3E4A4LVBAOwA96/JnWh0/UKH5rjJAaiVOYvCozdXf2kTOjdumamzJCL+MIUsNvfLtqiyOYtk/I/ZvFT6jMkWlIyNLQPdHQr1is7J416xa8lh8zALUT1VXu2u4XYU0wv/EElXwU3W1oxW1XS2RruZyyfmlyuuHpbu9SxYmKpcG9tzZHs389RcYn6rGOmhMZvVTfcc1m68ASLfhpE= madhu.pillai1@ibm.com
         - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCtAuDp3mu0C60gEpUB0Uj1OpApmQN/JIxd2B87hTX9NdMyOE0BWidhJlx/+BxGHWXQTInYpJbP4YOqM+ab8H9QmHngyoH9MFafKe+wTeUqe+ZNBz7O/PK2NZweE6eK/sQsqmxXlNFPt0/uhuw/lpCp1XfsuP2kHWI12AxC18zRSFQFDVUdYHADkLqX+jDIGoFSan8DFTTFIUI1j8vpt5UNcVXJJMwrl6TBZBs1cesdLMIHVfNcSgVZkE9bGDsJmoL/bVI0wnoqVJNQYBxjo/3Mz+CqAymZzzdI1gZ0nKYGwdy4SFw6sQgTgukyfb3D+3Wmiy9lC2p1/MCwocaBeZox2MR21Rouv6ljBUkMRhBPs2JG6IcMKIadyyzArliqxfiASvwd1d+Y22vH/jgRvnEIlbkvpAGMEX0LgjAcDW1KKKxCbSx9XYbkok53kDwQxYMbCzjr23woyaDbi7BTav1fSpG2jC9XBUXx3b8ygM2iCJwVXdDx9FLx4LbSRXZZAY0= nikita.dubrovskii@de.ibm.com
         - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDGCx0JhNfHB4FblCjnLVBFz3enROen5EINJOn2VFJRxiA2lXVAubcVga9+GyyADYf/Y76HRFOXmLS6qGqJBkmGK/5x50apZUQkKoLtsGpRHFXJAZ7DvVfyhlC7mtsSZzW+d81ZB1sCxPLrYzHpo5HPrhNqCP/hRRKK8OjlJtQRjup8suoZSCBkYf5+e2ROvqQR++p5ocYLYG/vX+8X0uhVhuwwbxqyXhLTHncEPtLAmVNMjs0PqHY4Oljh3Fc7JgskZcu+F6krP5Aono6W/zAz5bj/zgEg4OtvfjrfaP+ncW1n4BOPy+3VmZegukSYXt+fzOFgS2YSAnX30Q0jXm2hEzu0Xxz7tt8v1TZMT4XPOCV4ffmLfcXDDInyBUfEgxvTKIbWdjZMXDV5U4lCe8KUdDWqRaYLDSkCx5yWa2ICY63w3EueWERrOnTD/dsRRxBJqXfDEYU3FNcNA4LRFEI3+l7lxS5wZ5xoEvT9O1lghym+h4u/XrfYh6arl5BEr38= apiaseck@redhat.com
-        - ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIN4IzlQKAbfMEUsWrPG4JDJVCJDRHOC+U8ZbYRXC4K0p jbtrystram@redhat.com
+        - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCeVRo3murzVdV9OqZaDdn92Hor4hOejLlyY6AlydBv+CeXidz5Vdil7dBO6gX+QhbXusdKTDE7sb554C73ezssXyoC6ttfZE3j+bsVvy5BjlAgJ5jWEfBjR2/AJ1kjLhwg6fSroZpQHNmpPV3JOMKefRFG/yML6fFx6szGlxkOOesGCGMZTEEgM2m+TwyIEwb0bNdc/Qakir1kb/qOU2InfuDAX+gpjQVNYLSUbdiXQyqrCGFmeM0PhwgWmBJBrCLSO0a/QMtqnI/z9XRbjEGkkwQmVdDBDcxnO5mlqro5bTvAPKtbuM/pUjdOvMny1OQIgNFs5YJIMlKjQUoO/ZyVlndtldSDrE2L4hLOQu8i6Qx22FZKCco7h1yvCkJ0xlu4hKe4EqiueMScLjfYdA3IUH1e5Pi+F6v55Bcq6jxRUDTpbhFx94d6JznC8S40R/byvRAbsfvBS61XOtxEoA9orPooGBSW9AcP5ZTbSiAktQHUXih7XaUWIv7PnFMc60s= jbtrystram@redhat.com 
+>>>>>>> b568d8a (multiarch/add jbtrystram ssh jkey)
```

Comment:
Forgot to remove:
- >>>>>>> b568d8a (multiarch/add jbtrystram ssh jkey)

Reply:
wooops